st2api/st2api/controllers/v1/actionviews.py

```diff
@@ -98,32 +95,36 @@ class OverviewController(resource.ContentPackResourceController):
 
     include_reference = True
 
-    @jsexpose(arg_types=[str])
     def get_one(self, ref_or_id):
         """
             List action by id.
 
             Handle:
                 GET /actions/views/overview/1
         """
-        action_api = super(OverviewController, self)._get_one(ref_or_id)
-        return self._transform_action_api(action_api)
+        resp = super(OverviewController, self)._get_one(ref_or_id)
+        action_api = resp.json
+        result = self._transform_action_api(action_api)
+        resp.json = result
+        return resp
 
-    @jsexpose(arg_types=[str])
     def get_all(self, **kwargs):
         """
             List all actions.
 
             Handles requests:
                 GET /actions/views/overview
         """
-        action_apis = super(OverviewController, self)._get_all(**kwargs)
-        return map(self._transform_action_api, action_apis)
+        resp = super(OverviewController, self)._get_all(**kwargs)
+        action_apis = resp.json
+        result = map(self._transform_action_api, action_apis)
+        resp.json = result
+        return resp
 
     @staticmethod
     def _transform_action_api(action_api):
-        action_id = action_api.id
-        action_api.parameters = ParametersViewController._get_one(action_id).get('parameters')
+        action_id = action_api['id']
```

Comment:
Actually, I remembered why we returning Response from `._get_all()` and `._get_one()`. Rather fancy problem really.

We want to be able to set headers on the Response (in particular, 'X-Total-Count' and 'X-Limit'). And since we don't have `pecan.response` global for that anymore and we can't attach headers to model, we have to have a Request already present at that point.